packages/core/src/errors/types.ts

```diff
@@ -0,0 +1,8 @@
+import { BaseError } from "./base.js";
+
+export class InvalidContextSignatureError extends BaseError {
```

Comment:
same nit here, this is only relevant for the MultisigContext so we should move this into an errors namespace there